documentation/source/users/rmg/modules/isotopes.rst

```diff
@@ -82,3 +86,13 @@ The isotope generation script will output two files inside the nested folders
 RMG adjacency list structure. The other file of importance ``chem_annotated.inp``
 is a chemkin input file containing elements, species, thermo, and reactions of
 the entire system.
+
+Further reading & resources
+---------------------------
+
+A description of mechanisms generated using this module and its application to
+obtaining intramolecular isotopic enrichment information is described in
+Chemical Geology, Volume 514, page 1, doi: https://doi.org/10.1016/j.chemgeo.2019.02.036
```

Comment:
could you make these doi's clickable?
use this example format for rst: 
```
`Molpro <https://www.molpro.net/>`_
```

Reply:
fixed and checked to ensure it built properly with links